examples/simple-jsbeautifyrc/markdown/expected/yaml-front-matter.md

```diff
@@ -1,7 +1,8 @@
 ---
-layout: default
-title: this is my title
-description: this is my description
+layout:          default
+title:        this is my title
+description:     this is my description
 ---
 
-test ing
+ test
+   ing
```

Comment:
It looks like Remark simply did NOT beautify this file.
Original: https://raw.githubusercontent.com/kachkaev/atom-beautify/18e7f5ebad5f83e25a6a03e05ee15ada93329328/examples/simple-jsbeautifyrc/markdown/original/yaml-front-matter.md

```markdown
---
layout:          default
title:        this is my title
description:     this is my description
---

 test
   ing
```

Reply:
Yes, unfortunately remark does not format `frontmatter` and also does not collate the lines like in the example above. Shall we just remove this test for now?

Reply:
No, this would become a regression bug for Atom-Beautify. If our users assumed continued support for YAML front-matter formatting in their Markdown files, then switching the default will completely break their usage.

I'm leaning away from switching default to Remark unless we can beautify files as least as well as we had done previous with the default (TidyMarkdown). Next release should be a clear improvement, not a bug, to users.

Reply:
@wooorm do you think remark could solve this somehow?

Reply:
@kachkaev could you describe the possible breaking changes in the CHANGELOG? If we are going to switch from Tidy Markdown to Remark and cause all of these issues, we need to make it clear the cause and solution for users who do like their formatting looking like before.

Reply:
What is “frontmatter”? Here you show YAML, but there also exists TOML and some JSON front matter format as well.

Yes, remark could support this, through plugins, something like https://github.com/remarkjs/ideas/issues/3, and the linked issue contains some sample code, that should be made to format the YAML/TOML/JSON!